crates/iota-light-client/Cargo.toml

```diff
@@ -5,35 +5,54 @@ authors = ["IOTA Foundation <info@iota.org>"]
 edition = "2021"
 license = "Apache-2.0"
 publish = false
+description = "A light client for the IOTA blockchain"
 
 [lib]
 path = "src/lib.rs"
 
 [[bin]]
 name = "iota-light-client"
-path = "src/bin/light_client.rs"
+path = "src/main.rs"
+
+[[bin]]
+name = "generate-chk-snapshots"
```

Comment:
even if long, let's not abbreviate checkpoint

Reply:
Which script is using it?

Reply:
1. Yes, please create an issue
2. I would still rename it since we own it

Reply:
https://github.com/iotaledger/iota/pull/6135/commits/c923d95f0920e20dc3e1a57cd3aa6ce40a55adf9

Reply:
https://github.com/iotaledger/iota/issues/6825